app/src/main/java/de/westnordost/streetcomplete/overlays/street_furniture/StreetFurnitureOverlayForm.kt

```diff
@@ -0,0 +1,155 @@
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.LatLon
+import de.westnordost.streetcomplete.data.osm.mapdata.Node
+import de.westnordost.streetcomplete.databinding.FragmentOverlayStreetFurnitureBinding
+import de.westnordost.streetcomplete.osm.IS_DISUSED_STREET_FURNITURE_EXPRESSION
+import de.westnordost.streetcomplete.osm.IS_STREET_FURNITURE_INCLUDING_DISUSED_EXPRESSION
+import de.westnordost.streetcomplete.overlays.AbstractOverlayForm
+import de.westnordost.streetcomplete.util.getLocalesForFeatureDictionary
+import de.westnordost.streetcomplete.util.getLocationLabel
+import de.westnordost.streetcomplete.util.ktx.geometryType
+import de.westnordost.streetcomplete.util.ktx.viewLifecycleScope
+import de.westnordost.streetcomplete.view.controller.FeatureViewController
+import de.westnordost.streetcomplete.view.dialogs.SearchFeaturesDialog
+import kotlinx.coroutines.launch
+
+class StreetFurnitureOverlayForm : AbstractOverlayForm() {
+
+    override val contentLayoutResId = R.layout.fragment_overlay_street_furniture
+    private val binding by contentViewBinding(FragmentOverlayStreetFurnitureBinding::bind)
+
+    private var originalFeature: Feature? = null
+
+    private lateinit var featureCtrl: FeatureViewController
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val element = element
+        originalFeature = element?.let {
+            if (IS_DISUSED_STREET_FURNITURE_EXPRESSION.matches(element)) {
+                DummyFeature(
+                    "street_furniture/unknown_disused",
+                    requireContext().getString(R.string.unknown_disused_street_furniture),
+                    "ic_preset_maki_marker_stroked",
+                    element.tags
+                )
+            } else {
+                featureDictionary
+                    .byTags(element.tags)
+                    .forLocale(*getLocalesForFeatureDictionary(resources.configuration))
+                    .forGeometry(element.geometryType)
+                    .inCountry(countryOrSubdivisionCode)
+                    .find()
+                    .firstOrNull()
+                // if not found anything in the iD presets, then something weird happened
+                ?: DummyFeature(
+                    "street_furniture/unknown",
+                    requireContext().getString(R.string.unknown_street_furniture),
+                    "ic_preset_maki_marker_stroked",
+                    element.tags
+                )
+            }
+        }
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        // title hint label with name is a duplication, it is displayed in the UI already
+        setTitleHintLabel(element?.tags?.let { getLocationLabel(it, resources) })
+        setMarkerIcon(R.drawable.ic_quest_apple)
```

Comment:
I think a bench icon would be confusing here. So, should probably use a neutral white plus on a a green (amenity) background.

Reply:
You're not worried about it looking a lot like first aid?
https://en.wikipedia.org/wiki/First_aid#/media/File:ISO_7010_E003_-_First_aid_sign.svg

Maybe a tick or paste icon? Or a pin with a plus?

Reply:
No. If it is not thick enough, the plus will not look like that.

Reply:
The icon was changed to [`ic_preset_maki_marker_stroked`](https://commons.wikimedia.org/wiki/File:Maki7-marker-stroked.svg) in [`cbfa267`](https://github.com/streetcomplete/StreetComplete/pull/5373/commits/cbfa267cd25a33a7d12f57a17ef790ed603f8321).